pandas/core/ops.py

```diff
@@ -990,6 +991,93 @@ def _construct_divmod_result(left, result, index, name, dtype):
     )
 
 
+def dispatch_to_extension_op(left, right, op_name=None, is_logical=False):
+    """
+    Assume that left is a Series backed by an ExtensionArray,
+    apply the operator defined by op_name.
+    """
+
+    method = getattr(left.values, op_name, None)
+    deflen = len(left)
+    excons = type(left.values)._from_sequence
+    exclass = type(left.values)
+    testseq = left.values
```

Comment:
I'm having trouble understanding these names. (method makes sense).

Reply:
`excons` is the constructor used to construct a result from a sequence that has all its elements of type `ExtensionDtype`.  (so "ex" for "Extension" and "cons" for "constructor")  `exclass` is the underlying class of the `ExtensionArray` subclass.